index.js

```diff
@@ -0,0 +1,62 @@
+
+import { books, addNew, contact, titulo, autor} from "./modules/variables.js"
+import { containerBooks } from "./modules/list.js"
+import Libro from "./modules/list.js"
+import startTime from "./modules/clock.js";
```

Comment:
- [ ] I can see that you are trying to import some variables and the startTime form variables and clock js files, however, the variables.js and clock.js files are not present in your repository. Kindly add them

Reply:
Added clock.js and variables.js to repository.